shop/src/main/resources/application.properties

```diff
@@ -15,4 +15,8 @@ spring.flyway.url=jdbc:h2:~/online-shop-db
 spring.flyway.locations=filesystem:src/main/resources/db/migration
 
 #location finding strategy
-strategy="MULTIPLE_LOCATIONS_STRATEGY"
\ No newline at end of file
+strategy="MULTIPLE_LOCATIONS_STRATEGY"
+
+#security
+spring.main.allow-circular-references=true
```

Comment:
Don't do that. Circular dependencies are not allowed because they can get you into a big loopy problem. Try to solve your circular dependencies.